### PR TITLE
lando#2246: add new container reference, update docs and add unit test

### PR DESCRIPTION
**Pull request by rikvanderkemp**
- [X] My code includes the latest code from the `master` branch
- [x] My code includes an update to the [CHANGELOG](https://github.com/lando/lando/tree/master/docs/help)
- [X] My code includes a [unit test](https://docs.lando.dev/contrib/contrib-testing.html) if applicable
- [X] My code includes documentation updates if applicable.
- [ ] I have pinged [a project committer]

### CODE REVIEW

File: plugins/lando-services/services/mongo/builder.js [hunk 1]
```diff
@@ -8,8 +8,9 @@ module.exports = {
   name: 'mongo',
   config: {
     version: '4.0',
```
Comment (smutlord) [inline plugins/lando-services/services/mongo/builder.js:10]: @rikvanderkemp based on your comments on #2246 re: status of 4.0 and 4.1 does it make sense to set 4.2 as the default version?

Comment (rikvanderkemp) [reply plugins/lando-services/services/mongo/builder.js:10]: Great point! 

Especially when I look at the following page:
 https://www.mongodb.com/support-policy

4.0 and 4.2 are both TBD while 4.1 isn't mentioned.

4.0 is from June 2018 and 4.2 August 2019.

Based on this I would say set 4.2 to default.

Maybe even dropping 4.1 altogether and add a deprecation warning for it. @pirog did something similar to phpmyadmin.

What do you think?

Comment (pirog) [reply plugins/lando-services/services/mongo/builder.js:10]: @rikvanderkemp was looking at exactly the same. yes i think moving 4.1 into the "you can use it still but it throws a warning" group makes sense.

im merging a bunch of stuff together rn so im happy to make these changes

Comment (rikvanderkemp) [reply plugins/lando-services/services/mongo/builder.js:10]: Perfect that would be great.

**Comment by smutlord** [comment]
@rikvanderkemp ive got this over on this branch now. would love to get your review
https://github.com/lando/lando/pull/2251

**Pull request closed without merging**